src/Domain/Students/StudentGateway.php

```diff
@@ -141,7 +141,7 @@ public function queryStudentEnrolmentByRollGroup(QueryCriteria $criteria, $gibbo
         return $this->runQuery($query, $criteria);
     }
 
-    public function queryStudentsAndTeachersBySchoolYear(QueryCriteria $criteria, $gibbonSchoolYearID) 
+    public function queryStudentsAndTeachersBySchoolYear(QueryCriteria $criteria, $gibbonSchoolYearID, $gibbonRoleIDCurrentCategory) 
```

Comment:
Should the new param not be set to null by default? Or are there no current other use cases?

Reply:
Good call. There weren't any other uses that I could see, but that's a good habit to get in. Will make that change.